scripts/ts/common/ticket/jira/types.ts

```diff
@@ -0,0 +1,53 @@
+import * as t from "io-ts";
+
+const JiraIssueType = t.union([
+  t.literal("Epic"),
+  t.literal("Story"),
+  t.literal("Task"),
+  t.literal("Sottotask"),
```

Comment:
```suggestion
  t.literal("Subtaks"),
``` 
Is the usage of `Sottotask` mandatory to be compliant with Jira API data?

Reply:
Yes, because Jira doesn't have a fixed set of issue/ticket, but each project can have a custom set of `issuetype`, with all different `id`. In this case I've considered the field `name` of the `issueType`, using all the available default `issueType` names, but if in a project someone choose to use a new type of issue, this new kind will not be accepted.

```json
        "issuetype": {
            "self": "https://pagopa.atlassian.net/rest/api/3/issuetype/10049",
            "id": "10049",
            "description": "I sottotask monitorano parti di lavoro piccole e ben definite che fanno parte di un task più ampio.",
            "iconUrl": "https://pagopa.atlassian.net/secure/viewavatar?size=medium&avatarId=10316&avatarType=issuetype",
            "name": "Sottotask",
            "subtask": true,
            "avatarId": 10316,
            "entityId": "8703e710-31c2-4a50-9b6d-3bea6dd7a367",
            "hierarchyLevel": -1
        },

```